//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abceg_O_AVP.h
//
// Code generation for function 'abceg_O_AVP'
//

#ifndef ABCEG_O_AVP_H
#define ABCEG_O_AVP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abceg_O_AVP(double P_init, double V_init, double A_init, double P_wayp,
                 double V_wayp, double A_wayp, double A_max, double J_max,
                 double J_min, creal_T t[28]);

#endif
// End of code generation (abceg_O_AVP.h)
